python/tests/integration/completions/test_text_completion.py

```diff
@@ -37,16 +37,38 @@
 
 from semantic_kernel import Kernel
 from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.exceptions import ServiceInitializationError
 from tests.integration.completions.completion_test_base import CompletionTestBase, ServiceType
 from tests.integration.completions.test_utils import retry
 
 ollama_setup: bool = False
 try:
-    if os.environ["OLLAMA_MODEL"]:
-        ollama_setup = True
-except KeyError:
+    OllamaTextCompletion()
```

Comment:
Hmm, if we do this, it's very likely that we will never know if our integration tests are setup correctly since all of the ollama tests will be skipped instead of failing.

Reply:
Same for the other in chat & text completion